nilearn/datasets/func.py

```diff
@@ -1170,20 +1364,22 @@ def fetch_megatrawls_netmats(dimensionality=100, timeseries='eigen_regression',
     ----------
     dimensionality : int, optional
         Valid inputs are 25, 50, 100, 200, 300. By default, network matrices
-        estimated using Group ICA brain parcellations of 100 components/dimensions
-        will be returned. Default=100.
+        estimated using Group ICA brain parcellations
+        of 100 components/dimensions will be returned. Default=100.
 
     timeseries : str, optional
-        Valid inputs are 'multiple_spatial_regression' or 'eigen_regression'. By
-        default 'eigen_regression', matrices estimated using first principal
+        Valid inputs are 'multiple_spatial_regression' or 'eigen_regression'.
+        By default 'eigen_regression', matrices estimated using first principal
         eigen component timeseries signals extracted from each subject data
-        parcellations will be returned. Otherwise, 'multiple_spatial_regression'
+        parcellations will be returned.
+        Otherwise, 'multiple_spatial_regression'
         matrices estimated using spatial regressor based timeseries signals
         extracted from each subject data parcellations will be returned.
         Default='eigen_regression'.
 
     matrices : str, optional
-        Valid inputs are 'full_correlation' or 'partial_correlation'. By default,
+        Valid inputs are 'full_correlation' or 'partial_correlation'.
+        By default,
         partial correlation matrices will be returned otherwise if selected
         full correlation matrices will be returned.
```

Comment:
```suggestion
        By default, partial correlation matrices will be returned
        otherwise if selected full correlation matrices will be returned.
```